contracts/Raphael.sol

```diff
@@ -104,6 +104,10 @@ contract Raphael is ERC721Holder, Ownable, ReentrancyGuard {
         minVotesNeeded = 9644832 * 1e17; // 5% of initial distribution
     }
 
+    function getDidIVote(uint256 proposalIndex) public view returns (bool) {
```

Comment:
I'm also not sure about limiting this to `msg.sender` - another approach would be to take `(address voter, uint256 proposalIndex)` - that would seem to me to be the right approach

Reply:
I assumed it would be best to not allow anyone to check on whether others voted out of privacy concerns. I initially wrote it the way you suggested, but limited it intentionally.

Reply:
I renamed to match your suggestion.